test/App/Fossa/Configuration/ConfigurationSpec.hs

```diff
@@ -101,6 +106,26 @@ expectedVendoredDependencyFilters =
     , licenseScanPathFilterFileExclude = []
     }
 
+expectedLicenseGrepEntry :: ConfigGrepEntry
+expectedLicenseGrepEntry =
+  ConfigGrepEntry
+    { configGrepMatchCriteria = "(?i)proprietary"
+    , configGrepName = "Proprietary License"
+    }
+
+expectedLicenseSearch :: NonEmpty ConfigGrepEntry
+expectedLicenseSearch = NE.fromList [expectedLicenseGrepEntry]
+
+expectedKeywordGrepEntry :: ConfigGrepEntry
+expectedKeywordGrepEntry =
+  ConfigGrepEntry
+    { configGrepMatchCriteria = "\\w*(_token)"
+    , configGrepName = "token search"
+    }
```

Comment:
[nit, preference, optional] Prefer to use raw strings instead of escapes to make your intention more clear, e.g. via `QQ`: https://github.com/fossas/fossa-cli/blob/13cd70b073e3a26177e039ed290d08ddf6d6df96/integration-test/Analysis/LicenseScannerSpec.hs#L69-L71